docs/dev/apis/lethallib/customscrap.md

```diff
@@ -0,0 +1,94 @@
+---
+prev: false
+next: false
+description: A tutorial on how to add custom scrap items by using LethalLib.
+---
+# Custom scrap with LethalLib
+
+## Preamble
+Make sure you've done everything in the [LethalLib main page](/dev/apis/lethallib) first. This page will run through everything needed to make a new scrap item from scratch in the template project.
+
+At bare minimum, you will need the following added to your template project to create a custom scrap item:
+- A model of your scrap (and materials using the HDRP Lit shader)
+- SFX for picking up and putting down the scrap
+- An icon for the scrap when in the hotbar
+
+## Item Data
+The first thing you should create is the Item Data for your custom scrap, which can be created right-clicking in your asset window and going to create -> ScriptableObject -> Item
+![Image of the context menu in the asset browser of Unity, directing to Create -> ScriptableObjects -> Item](/images/lethallib/customscrap/CreateItem.png)
+
+This item data lets you configure basically everything about your item. There are a lot of options, but the important ones for scrap are:
+- **Item Name**: the name of the item.
+- **Spawn Position Types**: Spawn position types this item can spawn at. If the list is empty, the scrap can spawn anywhere (recommended to just leave this empty except for stuff like special scrap for custom moons).
+- **Two Handed**: Whether the item gives the "Hands Full" message and prevents changing items.
+- **Two Handed Animation**: Whether the item uses the two-handed animation for carrying or not.
+- **Weight**: A somewhat arbitrary number determing how heavy an item is. For reference, the Brush scrap is weight 1.1, the Cash Register is 1.8; pick a number that feels right.
+- **Item Spawns On Ground**: Should be true for scrap.
+- **Highest Sale Percentage**: Unknown; All items have this set to 80. Set it to that for safety.
+- **Is Conductive Metal**: Whether this item attracts lightning.
+- **Max/Min Value**: The value range for the item. Does not directly correlate to the range in game; a scalar is applied based on the moon it spawns on. For reference, the Big Bolt uses the range Max-80, Min-50.
+- **Spawn Prefab**: The prefab representing the item in the world. Creation of this is detailed later in this guide.
+- **Rotation/Position/Vertical Offset**: The offset values for holding the item in your hand. Mostly needs manual fiddling with until it looks right for a given item.
+- **Grab SFX, Drop SFX, Item Icon**: These must be assigned for the scrap to fully function.
```

Comment:
This is false, no exceptions will be raised the item will be fine it just won't make sound.